crates/build/re_types_builder/src/codegen/cpp/mod.rs

```diff
@@ -336,17 +336,21 @@ fn hpp_type_extensions(
         remaining_content = &remaining_content[end + COPY_TO_HEADER_END_MARKER.len()..];
     }
 
-    let comment = quote_comment(&format!(
-        "Extensions to generated type defined in '{}'",
+    let start_comment = quote_comment(&format!(
+        "START of extensions from {}:",
+        extension_file.file_name().unwrap()
+    ));
+    let end_comment = quote_comment(&format!(
+        "END of extensions from {}, start of generated code:",
```

Comment:
srsly why did this have to go into _this_ PR -.-

Reply:
Yeah -- if Emil was still here I would have asked to split that out.